ntbs-service/Pages/_HomepageKpiDetails.cshtml

```diff
@@ -3,17 +3,19 @@
 <div>
     @if (@Model.HomepageKpiDetails.Any())
     {
-        <h1 class="govuk-visually-hidden"> Homepage KPI Details </h1>
+        
         <filtered-homepage-kpi inline-template>
             <div>
                 <!-- Kpi details for non-JS browsers -->
                 <div class="homepage-kpi-container-without-js" id="kpi-details">
+                    <h1 class="kpi-heading"> KPIs </h1>

```

Comment:
How well does Screen reader deal with shorthands like KPI?
Would it be useful to provide `aria-label="Key Performance Indicators"`?